src/__tests__/modules-initialization-env/modules-initialization-env.test.ts

```diff
@@ -0,0 +1,32 @@
+import path from 'path';
+import { screen } from '@testing-library/react';
+import { getPage } from '../../../src';
+import { silenceConsoleError } from '../__utils__';
+
+silenceConsoleError('Text content did not match.');
```

Comment:
Is it expected behaviour that text content does not match? If so I think we should assert this and not just silence the error.